tests/std/tests/P0896R4_ranges_alg_is_permutation/test.cpp

```diff
@@ -142,7 +142,8 @@ struct instantiator {
 
                 NonCopyableBool()                       = default;
                 NonCopyableBool(const NonCopyableBool&) = delete;
-            } b;
+            };
+            NonCopyableBool b;

```

Comment:
Tragically missed opportunity to name this type `NonCopyabool`. (No change requested.)